packages/data-store/src/connection/ConnectionStore.ts

```diff
@@ -133,7 +145,7 @@ export class ConnectionStore extends AbstractConnectionStore {
       return Promise.reject(Error(`No party found for id: ${partyId}`))
     }
 
-    if (!this.hasCorrectConfig(connection.type, connection.config)) {
+    if (!this.hasCorrectConfig(connection.type, connection.config)) { // TODO fix
```

Comment:
Fix should have a description ... or This task should be completed and comment removed.

Reply:
removed